src/components/TabsFile/index.tsx

```diff
@@ -228,10 +229,11 @@ export default function TabsFile({
         BigInt(quoteResult.tokenAmount) + BigInt(50000000000000000)
 
       setQuoteWithBuffer(String(newQuoteFee))
-
       // Check if user has wrapped matic in their wallet
       if (wmaticBalance < newQuoteFee) {
         console.log('User does not have enough wMatic')
+        setErrorUpload(true)
```

Comment:
Having no WMATIC isn't an upload error, it's part of the normal user flow now. We shouldn't show an error message for this. If we show any message it should be an info message